apps/web/components/IncreasePower.tsx

```diff
@@ -86,12 +91,10 @@ export const IncreasePower = ({
     registerTokenDecimals,
   );
 
-  const chainId = getChainIdFromPath();
-
   const { data: accountTokenBalance } = useBalance({
     address: connectedAccount,
     token: registerToken as `0x${string}` | undefined,
-    chainId: chainId || 0,
+    chainId: Number(chainId),
```

Comment:
already number not need cast

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/48ab8c83d97f1a0865d21861c3dc24e7f9d5a821

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/5496912036e05bbb3274af4d9c822c308e3b2946